ClientSide(FrontEnd)/Provider_Side/HomePageProvider.html

```diff
@@ -1,12 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-<body>
-    <h1>Provider Side Home Page</h1>
-</body>
-</html>
```

Comment:
add one empty line at the end of your code

Reply:
homepage.html is a unnecessary file i deleted this